backend/homework/2/src/main/java/QuestionTwo/SentimentAnalyzer.java

```diff
@@ -0,0 +1,123 @@
+package QuestionTwo;
+
+import org.slf4j.LoggerFactory;
+import org.slf4j.Logger;
+
+import java.util.Arrays;
+
+public class SentimentAnalyzer {
```

Comment:
Try to break down a GOD class into multiple classes. Try using Singularity principle for SOLID.

Reply:
will try to do modular classes next time.